libopenage/util/fslike/directory.cpp

```diff
@@ -37,7 +38,7 @@
 #include "../misc.h"
 #include "../path.h"
 
-
+namespace fs = std::filesystem;
```

Comment:
Can we  not create an alias here and spell out the full namespace instead? I think it will be more readable this way.

Reply:
Yes, probably the names won't become too long anyway.